birthday-pair-programming.js

```diff
@@ -9,38 +9,74 @@ const gameGroup = [
       first: "Anna",
       last: "Brekke",
     },
-  },
+    init: function() {
+      this.age = getAge(this.birthday);
+      delete this.init;
+      return this;
+    }
+  }.init(),
   {
     birthday: "February 27, 1990",
     name: {
       first: "Andrew",
       last: "Brenner",
     },
-  },
+    init: function() {
+      this.age = getAge(this.birthday);
+      delete this.init;
+      return this;
+    }
+  }.init(),
   {
     birthday: "July 22, 1989",
     name: {
       first: "Felicia",
       last: "Brenner",
     },
-  },
+    init: function() {
+      this.age = getAge(this.birthday);
+      delete this.init;
+      return this;
+    }
+  }.init(),
   {
     birthday: "December 8, 1987",
     name: {
       first: "Dan",
       last: "Richmond",
     },
-  },
+    init: function() {
+      this.age = getAge(this.birthday);
+      delete this.init;
+      return this;
+    }
+  }.init(),
   {
     birthday: "January 30, 1989",
     name: {
       first: "Justin",
       last: "Veres",
     },
-  },
+    init: function() {
+      this.age = getAge(this.birthday);
+      delete this.init;
+      return this;
+    }
+  }.init()
 ];
 
 // FUNCTIONS
+  function getAge(birthdate) {
+  const birthday = new Date(birthdate).getTime();
+  console.log(birthdate);
+  console.log(birthday);
```

Comment:
Can you go ahead and remove these two console.logs? I think they are polluting your console when you want it to be easy to find the table.